qiskit/providers/ibmq/api_v2/session.py

```diff
@@ -14,19 +14,22 @@
 
 """Session customized for IBM Q access."""
 
+import os
 from requests import Session, RequestException
 from requests.adapters import HTTPAdapter
 from urllib3.util.retry import Retry
 
 from .exceptions import RequestsApiError
+from ..version import __version__ as ibmq_provider_version
 
 STATUS_FORCELIST = (
     500,  # Internal Server Error
     502,  # Bad Gateway
     503,  # Service Unavailable
     504,  # Gateway Timeout
 )
-CLIENT_APPLICATION = 'qiskit-api-py'
+CLIENT_APPLICATION = 'ibmqprovider/version-' + ibmq_provider_version
```

Comment:
Can you change it to `ibmqprovider/1.2.3.4`? The rationale would be that this way "parsers" can just split by `/` to get the desired fields.